lib/idx/util.ts

```diff
@@ -35,6 +35,24 @@ export function findRemediationByName(idxRemediation: IdxRemediation, name: stri
   });
 }
 
+// Return first match idxRemediation in allowed remediators for flow with authenticators
+export function getIdxRemediationWithAuthenticators(remediators, idxRemediations, requestedAuthenticators) {
```

Comment:
The logic should be handled here: https://github.com/okta/okta-auth-js/blob/414d65eff70b1b0738c70b921d7494db0024a4d8/lib/idx/remediators/SelectAuthenticator.ts#L12-L25